_projects/git_task_list.md

```diff
@@ -0,0 +1,118 @@
+different organization use different kind of issue tracking such as Phabricator
+and GitLab.
+
+**Git Task List** is conceptually the same as [GSOC prep task](http://summerofcode.github.io/gsoc-prep-tasks).
+Git Task List target a broader task type which include pull request review as
+the task. The goal of this project is to help developer to keep contributing to
+open source project by providing a sets of tasks that has been filtered using
+predefined keywords.
+
+In coala, this project will help sharing the tasks of reviewing a pull request
+on a project accross GitHub and GitLab. It will help fix the problem where
+developer spend too much time finding PR that haven’t been reviewed. This
+projects aims to getting more developer doing reviews works and sharing reviews
+workload, by providing an easy way to find issues that has not been reviewed
+both on http://github.com/coala and http://gitlab.com/coala.
+
+This will also help engage the developer to keep contributing to a project where
+they can’t find any issue that they can fix or keep the newcomer issue for new
+comer as learning path and developer with more experience can contribute by
+reviewing newcomer’s work PR.
+
+The goal of this projects is a static-site javascript web page where the user
+can find Pull Request that haven’t been reviewed. User can select projects in
+Open Source hosting services like GitHub or GitLab. They also have the ability
+to select prefered programming language, tags, owner of the pull requests, and
+other metadata that is provided by the web services (GitHub/GitLab).
+
+Because the concept of this project, is the same as [GSOC prep task](http://summerofcode.github.io/gsoc-prep-tasks), it can be used as the starting point of this project.
+
+#### Known Issue Tracker Usage In GSOC Organization
+
+Issue Tracker | Usage
+--- | ---
+GitHub | 16
+Trac | 7
+Atlassian Jira | 7
+GitLab | 4
+phabricator | 4
+BugZilla | 4
+Usabilla | 2
+Redmine | 2
+
+Because of the difficulty to support all of the known issue tracker in the range
+of GSOC timeline. This project will provide support for GitHub, GitLab, and
+Phabricator.
+
+#### Identifying Task Of A Specific Organization
+
+For a hosted issue tracker (like GitHub.com & GitLab.com) it is already
+documented and we can confidently know what resource are posible and what is not
+. The process of adding support for a new organization is posible using an
+external data source such as [Wikidata](http://wikidata.org). The app can fetch
+for list of organization that use GitHub / GitLab via [Qikidata Query API](https://query.wikidata.org/)
+
+The case is different for a self hosted Organization. The accessibility is
+different. For organization that use Phabricator as issue tracker, the process
+of adding support to it will be manually for each organization to make sure that
+the API is accessible.
+
+#### Milestone
+
+##### Phase 1
+- Support adding user token
```

Comment:
This is not a feature, and should not be listed here.
c.f. https://github.com/coala/projects/pull/579#discussion_r190102038

I am being very pedantic about this as I do not want this project to be doing anything like https://github.com/summerofcode/gsoc-prep-tasks/issues/15 , which is cool but not in scope and will be an enormous time drain (especially when getting it to work on GitLab, Phabricator, etc,etc), and that type of functionality has been put into the `gh-board` meta-review project so it is not needed or wanted by coala in the git-tasks project, as it could potentially even cause problems for the meta-review project if there are two tools which allow that type of user functionality.

Reply:
@jayvdb removed